component_sdk/python/kfp_component/google/dataflow/_client.py

```diff
@@ -30,15 +30,19 @@ def launch_template(self, project_id, gcs_path, location,
         ).execute()
 
     def get_job(self, project_id, job_id, location=None, view=None):
-        return self._df.projects().jobs().get(
+        if not location:
+            location = 'us-central1'
```

Comment:
Could we move the default location to somewhere that the get_job and cancel_job can share? For better maintenance.

Reply:
done